core/domain/general_feedback_services.py

```
@@ -155,6 +158,51 @@ def _determine_destination_dashboard(
+
+    Raises:
+        ValueError. The feedback does not belong to the requested dashboard.
+    """
+    if dashboard == feconf.DESTINATION_CREATOR:
+        if (
+            feedback.destination_dashboard != feconf.DESTINATION_CREATOR
+            or feedback.lesson_metadata is None
+        ):
+            raise ValueError(
+                'Feedback does not belong to the requested dashboard.'
+            )
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)